client/src/components/IndicatorsViz.js

```diff
@@ -305,7 +305,7 @@ class IndicatorsVizImplementation extends Component {
             } else if (timeSpan === "month") {
               // Make date value include day:
               var fullDate = this._data.labels[tooltipItem[0].index].concat("-15");
-              return Helpers.formatDateForTimeline(fullDate, locale);
+              return Helpers.formatDate(fullDate, { year: "numeric", month: "long" }, locale);
```

Comment:
Same as above— it might make sense to save the options object as a variable somewhere, as you use it twice. If you choose to do so, you might want to name it something specific as we do seem to use other date options within this component, like `{ month: "short" }`, and would want to be clear about that distinction.